articles/portal/ptl-ref-security-settings.md

```diff
@@ -0,0 +1,60 @@
+---
+title: UKCloud Portal security settings | UKCloud Ltd
+description: Provides information about the various security settings available in the UKCloud Portal, including memorable word, two-factor authentication (2FA) and IP restrictions
+services: portal
+author: Sue Highmoor
+reviewer:
+lastreviewed: 07/11/2019
+
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: UKCloud Portal security settings
+toc_fullpath: Reference/ptl-ref-security-settings.md
+toc_mdlink: ptl-ref-security-settings.md
+---
+
+# UKCloud Portal security settings
+
+The UKCloud Portal provides various settings to enable you to provide additional security to your accounts.
+
+![Portal security settings](images/ptl-security-settings.png)
+
+> [!TIP]
+> A green tick next to a section indicates that particular security setting has been enabled.
+
+![Enabled security setting](images/ptl-security-settings-enabled.png)
+
+- **Password expiration** - Set passwords to expire after a specified number of days. This ensures that users change their passwords frequently. For more information, see [*How to set up password expiration*](ptl-how-setup-password-expiration.md).
+
+- **Memorable word authentication** - Add an extra level of authentication whereby users provide letters from a memorable word as well as their password to confirm their identity. For more information, see [*How to enable memorable word authentication*](ptl-how-enable-memorable-word.md).
+
+- **Google two-factor authentication (2FA)** - Add an extra level of authentication whereby users provide a security code from an authenticator app (such as Google Authenticator or Authy) as well as their password to confirm their identity. For more information, see [*How to enable two-factor authentication in the UKCloud Portal*](ptl-how-setup-2fa.md).
+
+- **Portal access restriction by IP address** - Restrict access so that specific IP addresses are unable to log in to the Portal. For more information, see [*How to restrict access to the Portal via IP address*](ptl-how-restrict-ip-access-portal.md).
+
+- **vCloud Director API access restriction by IP address** - Restrict access so that specific IP addresses are unable to authenticate with the vCloud API. For more information, see [*How to restrict access to the vCloud API via IP address*](ptl-how-restrict-ip-access-vcloud-api.md).
```

Comment:
Restrict access so that only specific IP addresses are able to authenticate with the vCloud API.

Reply:
Fixed